process_basic.go

```diff
@@ -169,7 +169,7 @@ func (p *basicProcess) Wait(ctx context.Context) (int, error) {
 
 	select {
 	case <-ctx.Done():
-		return -1, errors.New("operation canceled")
+		return -1, errors.Wrap(ctx.Err(), "operation canceled")
```

Comment:
Probably can just return `ctx.Err()`

Reply:
I changed the wrapping message. Jasper processes use different contexts everywhere so it's hard to trace which context is the source of the error without some error wrapping (e.g. distinguishing if it was the wait method's context that errored, the `exec.Cmd`'s context that errored, or the background wait goroutine's context that errored).